certification/internal/policy/container/has_modified_files.go

```diff
@@ -30,7 +30,7 @@ type packageFilesRef struct {
 func (p *HasModifiedFilesCheck) Validate(ctx context.Context, imgRef certification.ImageReference) (bool, error) {
 	packageFiles, err := p.getDataToValidate(ctx, imgRef)
 	if err != nil {
-		return false, fmt.Errorf("%w: %s", errors.ErrExtractingTarball, err)
+		return false, fmt.Errorf("%s", err)
```

Comment:
This looks like it needs a wrapping error.

Reply:
I'm trying to take away the wrapping, because the error is not always an `ErrExtractingTarball` error. Maybe it just makes more sense to return err here.

Reply:
I think the right thing to do here is to wrap with an appropriate error, relevant to the type of check (e.g. ErrDetectingModifiedFiles).

Reply:
I added an error and wrapped the error that bubbles up.